src/layers/core/scatterplot-layer/scatterplot-layer.js

```diff
@@ -65,9 +72,47 @@ export default class ScatterplotLayer extends Layer {
       instanceRadius: {size: 1, accessor: 'getRadius', defaultValue: 1, update: this.calculateInstanceRadius},
       instanceColors: {size: 4, type: GL.UNSIGNED_BYTE, accessor: 'getColor', update: this.calculateInstanceColors}
     });
+
+    if (this.props.fp64) {
+      this.state.attributeManager.addInstanced({
+        instancePositions64xyLow: {size: 2, accessor: 'getPosition', update: this.calculateInstancePositions64xyLow}
+      });
+    }
     /* eslint-enable max-len */
   }
 
+  updateModel({props, oldProps, changeFlags}) {
+    if (props.fp64 !== oldProps.fp64) {
+      const {gl} = this.context;
+      this.setState({model: this._getModel(gl)});
+    }
+  }
+
+  updateAttribute({props, oldProps, changeFlags}) {
+    if (props.fp64 !== oldProps.fp64) {
+      const {attributeManager} = this.state;
+
+      // all attributes needs to be set again after the model changes
```

Comment:
While I assume that you are correct, this does not sound like the way the attribute manager was intended this to work. Let's discuss.